dynalab/tasks/annotation_mock_data.py

```diff
@@ -4,7 +4,7 @@
 
 
 def get_source_data(annotation, name_to_annotation_dict):
-    source_reference_name = annotation["constructor_args"]["reference_name"]
+    source_reference_name = annotation["reference_name"]
```

Comment:
Is it possible for `reference_name` to not be in the particular annotation (I'm still not one hundred percent clear on how these configs work, so might be dumb question)

Reply:
I guess the line deleted did not check if `reference_name `as not in `annotation["constructor_args"]` , so this is probably not going to cause a problem

Reply:
My understanding is that get_source_data is a helper function specifically for types that have reference_name. I think it should be ok.